apps/client/src/app/screens/signin/signin.tsx

```diff
@@ -0,0 +1,47 @@
+import { useContext } from 'react';
+import { TextInput, Button, Group, Center } from '@mantine/core';
+import { useForm } from '@mantine/form/';
+import { AuthContext } from '../../contexts/auth-provider/auth-provider';
+
+/* eslint-disable-next-line */
+export interface ISigninProps {}
+
+export function Signin(props: ISigninProps) {
+  const setUser = useContext(AuthContext);
```

Comment:
Please use object destructuring, and use hook that you wrote for auth context
```suggestion
  const { signIn } = useAuthContext()
```